### ISSUE TITLE
call(string) Deprecation

### ISSUE THREAD

**Issue by marcus-n3rd** [oddbird/accoutrement-color#10]
I just recently started experiencing these warnings with a fresh node_modules install.

```
DEPRECATION WARNING: Passing a string to call() is deprecated and will be illegal
in Sass 4.0. Use call(get-function("color")) instead.

DEPRECATION WARNING: Passing a string to call() is deprecated and will be illegal
in Sass 4.0. Use call(get-function("size")) instead.
```

**Comment by mirisuzanne** [comment]
That comes from a change in Sass 3.5+, and is being fixed: https://github.com/oddbird/accoutrement-color/commit/483ffa7c0a3c9761b9739d457403e97814b8f303#commitcomment-28323266

**Comment by marcus-n3rd** [comment]
Sweet. Thank you! Just making sure it was on the radar.

**Comment by mirisuzanne** [comment]
Improved fix in 51dccd4 allows you to capture functions in a shared map that we can use for lookup:

```scss
$functions: (
  'darken': get-function('darken'),
  'lighten': get-function('lighten'),
);
```

That way, you shouldn't have to make any changes to your existing color maps.
